react/features/app/middlewares.web.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-
+import '../authentication/middleware';
```

Comment:
We now have this in both web and mobile right? Then you can move it to middlewares.any.js

Reply:
Middleware.web is not the same with native, I suggest we don't do that.

Reply:
Right.